lfucg-exactions/client/js/components/PlatForm.js

```diff
@@ -377,16 +391,16 @@ class PlatForm extends React.Component {
                                                     {platZoneDuesList}
                                                 </div>
                                             </div>
-                                            <div className="col-xs-12 dues-table-heading"><h3>Plat Dues</h3></div>
+                                            <div className="col-xs-12 dues-table-heading"><h3>Plat Exactions</h3></div>
                                             <div className="row">
                                                 <div className="col-sm-6">
-                                                    <FormGroup label="* Sewer Fees Due" id="sewer_due">
-                                                        <input type="number" className="form-control" placeholder="Sewer Fees Due" />
+                                                    <FormGroup label="* Sewer Exactions" id="sewer_due">
```

Comment:
Are these * still supposed to be here? I noticed them removed from the other labels.